dashboard/client/components/item-object-list/item-list-layout.tsx

```diff
@@ -114,9 +114,7 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     const { store, dependentStores, isClusterScoped } = this.props;
     const stores = [store, ...dependentStores];
     if (!isClusterScoped) stores.push(namespaceStore);
-    const loadingStores = stores.filter(store => !store.isLoaded);
-    loadingStores.forEach(store => store.loadAll());
-    await when(() => loadingStores.every(store => !store.isLoading));
+    await Promise.all(stores.map(store => store.loadAll()));
```

Comment:
Do you want to load resources every time on visiting some page? Clusters might be huge as you know...

Reply:
Not sure we have other options. If we don't load them every time list is loaded we are actually using stale store where we replay all events since last visit to the same list. This might cause list to "jitter" while events are replayed. Worst case is that kube apiserver does not have all events and then replay is only partial -> list goes out-of-sync.